code/modules/food/recipes_microwave.dm

```diff
@@ -18,7 +18,7 @@ I said no!
 	)
 	result = /obj/item/reagent_containers/food/snacks/telebacon
 */
-
```

Comment:
This file as a whole need an Occulus Edit start / Occulus Edit End just so it don't get overridden by accident

Reply:
Replaced this, the file is reenabled and the entire change is modular.